travis/test_server.py

```diff
@@ -296,11 +304,33 @@ def main(argv=None):
     counted_errors = 0
     for to_test in to_test_list:
         print("\nTesting %s:" % to_test)
-        subprocess.call(["createdb", "-T", dbtemplate, database])
+        db_odoo_created = False
+        try:
+            db_odoo_created = subprocess.call(
+                ["createdb", "-T", dbtemplate, database])
+        except subprocess.CalledProcessError:
+            db_odoo_created = True
         for command, check_loaded in commands:
-            command[-1] = to_test
-            # Run test command; unbuffer keeps output colors
-            command_call = ["unbuffer"] + command
+            if db_odoo_created and instance_alive:
+                # If exists database of odoo test
+                # then start server with regular command without tests params
+                rm_items = [
+                    'coverage', 'run', '--stop-after-init',
+                    '--test-enable', '--init', None,
+                    '--log-handler', 'openerp.tools.yaml_import:DEBUG',
+                ]
+                command_call = [item
+                                for item in commands[0][0]
+                                if item not in rm_items] + \
+                    ['--db-filter=^%s$' % database]
+            else:
+                command[-1] = to_test
+                if is_runbot:
+                    command_call = []
```

Comment:
The file of runbot is a text plain in a browser.
It when I used with `unbuffer` I saw next logfile:
`\033[1;32mINFO\033[0;m openerp_test`
But runbot use [regex](https://github.com/odoo/odoo-extra/blob/1e2d4bfdfe0e66e0a000e10edd8bf68b756eff46/runbot/runbot.py#L43-L44) without color format